scripts/cleanup-github-branches.sh

```
@@ -0,0 +1,202 @@
+# Protected branches that should never be deleted
+PROTECTED_BRANCHES=("main" "dev" "staging")
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+CYAN='\033[0;36m'
+NC='\033[0m' # No Color
+
+usage() {
```

Shellcheck (0.11.0)

[error] Couldn't parse this brace group. Fix to allow more checks.

(SC1073)